docs/src/to-contribute/boterinas.md

```diff
@@ -0,0 +1,60 @@
+# Bot for Asterinas 
```

Comment:
Remove "Asterinas" from the title. Every page of the Asterinas book is for Asterinas. So mentioning Asterinas in the title provides little information. I think good candidate names for this page are "Boterinas" and "Bot for Github".

Reply:
I prefer "Boterinas"~